doc/classes/PackedScene.xml

```diff
@@ -42,6 +47,8 @@
 	</methods>
 	<members>
 		<member name="_bundled" type="Dictionary" setter="_set_bundled_scene" getter="_get_bundled_scene">
+			A [class Dictionary] representation of the scene contents.
```

Comment:
Should be `[Dictionary]`, though here it's already clear and clickable in the member signature, so I think just using the plain English "dictionary" would be better.